### PR TITLE
Facu - PODVI - Adds aimbar logic, enhanced backgrounds and tileset

### PR DESCRIPTION
**Pull request by fmestevez**

### CODE REVIEW

**Comment by fmestevez** [comment]
@hseghetti I wasn't able to deploy the code, look below:

 !  Your account fmestevez@gmail.com does not have access to podgame.
 !  
 !  SSH Key Fingerprint: 03:19:50:90:a1:59:6c:62:dc:11:8d:78:ca:07:20:0b

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

File: public/assets/pista1.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
...2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3],
```
Comment (Marthinoth) [inline public/assets/pista1.json:4]: What does this mean? just curious (data)

File: public/assets/pista1.tmx [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
-2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```
Comment (Marthinoth) [inline public/assets/pista1.tmx:8]: is this, right? the data you have in pista.json. Gotcha

File: public/js/menu.js [hunk 3]
```diff
@@ -18,8 +20,8 @@ var menuState = {
 		}
 
 		// Start the game when the up arrow key is pressed
-		var upKey = game.input.keyboard.addKey(Phaser.Keyboard.UP);
-		upKey.onDown.addOnce(this.start, this);
+		var spaceKey = game.input.keyboard.addKey(Phaser.Keyboard.SPACEBAR);
```
Comment (Marthinoth) [inline public/js/menu.js:23]: Thanks! It was ugly to start with the up key :)

File: public/js/play.js [hunk 4]
```diff
@@ -17,15 +19,28 @@ var playState = {
...
+        this.aimbar.anchor.setTo(0.5, 0.5);
+        this.aimbar.fixedToCamera = true;
+        this.aimpointer = game.add.sprite(14, 20, 'aimpointer');
+        this.aimpointer.anchor.setTo(0.5, 0.5);
+        this.aimpointer.fixedToCamera = true;
+        this.setAimpointerAnimation();
 	},
 
-	update: function() {
+	update: function() {
```
Comment (Marthinoth) [inline public/js/play.js:43]: Try to have the IDE well configured. It is your friend and allows us (the reviewers) to focus on defects in the code

**Comment by Marthinoth** [comment]
Looks good for me

File: public/assets/pista1.tmx (hunk 2 above)
Comment (fmestevez) [inline public/assets/pista1.tmx:8]: Yes, this is a matrix, with each number representing the index of a tile on the tileset

File: public/js/play.js (hunk 4 above)
Comment (fmestevez) [inline public/js/play.js:43]: Actually that was a loose tab, I'll delete it on the next push. Thanks for the heads up.

**Pull request merged**